### ISSUE TITLE
No valid host for ARM64 node when launching instance in VEXXHOST

### ISSUE THREAD

**Issue by kiwik** [theopenlab/openlab#62]
Launching VEXXHOST ARM instance failed in OpenLab nodepool, Error message is **{'code': 500, 'message': 'No valid host was found. There are not enough hosts available.', 'created': '2018-05-28T07:05:12Z'}**. So I have to set min-ready of label **ubuntu-bionic-vxh-arm64** to 0, we can enable it when the issue is addressed. See details:

```bash
(openstack) server list
+--------------------------------------+---------------------------------------------+--------+--------------------------------------+--------------------------+---------------+
| ID                                   | Name                                        | Status | Networks                             | Image                    | Flavor        |
+--------------------------------------+---------------------------------------------+--------+--------------------------------------+--------------------------+---------------+
| 79961a25-2671-40d5-9d07-35359f41fd00 | ubuntu-bionic-vxh-arm64-vexxhost-0000006979 | ERROR  |                                      |                          | v2-highcpu-8  |
| bf5398cd-a986-4261-b442-86f886b18926 | ubuntu-bionic-vxh-arm64-vexxhost-0000006979 | ERROR  |                                      |                          | v2-highcpu-8  |
| 29d46639-749c-4ba7-9344-74cabde176ea | ubuntu-bionic-vxh-arm64-vexxhost-0000006979 | ERROR  |                                      |                          | v2-highcpu-8  |
| d479c63b-97a5-45ce-89f4-99461e300202 | ubuntu-bionic-vxh-arm64-vexxhost-0000006978 | ERROR  |                                      |                          | v2-highcpu-8  |
| 5f88dabc-867a-4630-8338-82621c62d172 | ubuntu-bionic-vxh-arm64-vexxhost-0000006978 | ERROR  |                                      |                          | v2-highcpu-8  |
| 6492332d-1325-48c7-8901-3263f954b91c | ubuntu-bionic-vxh-arm64-vexxhost-0000006978 | ERROR  |                                      |                          | v2-highcpu-8  |
| 812d331b-21de-46bd-889b-7425057e3ac0 | ubuntu-bionic-vxh-arm64-vexxhost-0000006977 | ERROR  |                                      |                          | v2-highcpu-8  |
| 5fe7a2be-8c27-4702-a23e-b51ad6c1db29 | ubuntu-bionic-vxh-arm64-vexxhost-0000006977 | ERROR  |                                      |                          | v2-highcpu-8  |
| c545e9df-15b6-4ed4-a313-421bc32f448e | ubuntu-bionic-vxh-arm64-vexxhost-0000006977 | ERROR  |                                      |                          | v2-highcpu-8  |
| d31ebb87-f3ca-4a13-ae93-d4885a9ec8f0 | ubuntu-xenial-vexxhost-vexxhost-0000006951  | ACTIVE | int-net=192.168.0.24, 162.253.55.103 | ubuntu-xenial-1527478434 | v1-standard-8 |
| a9e935fe-ca10-4e8e-9f0b-f2e0bedaa18a | ubuntu-xenial-vexxhost-vexxhost-0000006950  | ACTIVE | int-net=192.168.0.27, 162.253.55.231 | ubuntu-xenial-1527478434 | v1-standard-8 |
| 4e1d64f6-de6f-4618-a38d-b845e392e5c1 | ubuntu-xenial-vexxhost-vexxhost-0000006949  | ACTIVE | int-net=192.168.0.23, 162.253.55.78  | ubuntu-xenial-1527478434 | v1-standard-8 |
| 648f5bfb-2e88-4725-97d9-838022968340 | ubuntu-xenial-vexxhost-vexxhost-0000006948  | ACTIVE | int-net=192.168.0.31, 162.253.55.199 | ubuntu-xenial-1527478434 | v1-standard-8 |
| f3d76f11-a8fa-4b50-9150-ee2e7ed8800a | ubuntu-xenial-vexxhost-vexxhost-0000006945  | ACTIVE | int-net=192.168.0.18, 162.253.55.163 | ubuntu-xenial-1527478434 | v1-standard-8 |
| 64303ef5-084a-47db-9d64-91f8381e3a0b | ubuntu-xenial-vexxhost-vexxhost-0000006943  | ACTIVE | int-net=192.168.0.20, 162.253.55.220 | ubuntu-xenial-1527478434 | v1-standard-8 |
| 8f0b953b-a0fa-4f07-870f-5e8bad48012d | ubuntu-xenial-vexxhost-vexxhost-0000006942  | ACTIVE | int-net=192.168.0.13, 162.253.55.12  | ubuntu-xenial-1527478434 | v1-standard-8 |
+--------------------------------------+---------------------------------------------+--------+--------------------------------------+--------------------------+---------------+
(openstack) server show d479c63b-97a5-45ce-89f4-99461e300202
+-----------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| Field                       | Value                                                                                                                                                                                                                             |
+-----------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| OS-DCF:diskConfig           | MANUAL                                                                                                                                                                                                                            |
| OS-EXT-AZ:availability_zone | ca-ymq-2                                                                                                                                                                                                                          |
| OS-EXT-STS:power_state      | NOSTATE                                                                                                                                                                                                                           |
| OS-EXT-STS:task_state       | None                                                                                                                                                                                                                              |
| OS-EXT-STS:vm_state         | error                                                                                                                                                                                                                             |
| OS-SRV-USG:launched_at      | None                                                                                                                                                                                                                              |
| OS-SRV-USG:terminated_at    | None                                                                                                                                                                                                                              |
| accessIPv4                  |                                                                                                                                                                                                                                   |
| accessIPv6                  |                                                                                                                                                                                                                                   |
| addresses                   |                                                                                                                                                                                                                                   |
| config_drive                |                                                                                                                                                                                                                                   |
| created                     | 2018-05-28T07:05:11Z                                                                                                                                                                                                              |
| fault                       | {'code': 500, 'message': 'No valid host was found. There are not enough hosts available.', 'created': '2018-05-28T07:05:12Z'}                                                                                                     |
| flavor                      | v2-highcpu-8 (221de281-95ec-414f-8e42-c86c9e0b318d)                                                                                                                                                                               |
| hostId                      |                                                                                                                                                                                                                                   |
| id                          | d479c63b-97a5-45ce-89f4-99461e300202                                                                                                                                                                                              |
| image                       |                                                                                                                                                                                                                                   |
| key_name                    | openlab-zuul-key                                                                                                                                                                                                                  |
| name                        | ubuntu-bionic-vxh-arm64-vexxhost-0000006978                                                                                                                                                                                       |
| project_id                  | 69b5f56c11e84927abebfdca8039b48b                                                                                                                                                                                                  |
| properties                  | groups='vexxhost,ubuntu-bionic-vxh-arm64,ubuntu-bionic-vxh-arm64', nodepool_image_name='ubuntu-bionic-vxh-arm64', nodepool_node_id='0000006978', nodepool_node_label='ubuntu-bionic-vxh-arm64', nodepool_provider_name='vexxhost' |
| status                      | ERROR                                                                                                                                                                                                                             |
| updated                     | 2018-05-28T07:05:12Z                                                                                                                                                                                                              |
| user_id                     | f89a11b823e2496394de499c1a3d9322                                                                                                                                                                                                  |
| volumes_attached            |                                                                                                                                                                                                                                   |
+-----------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
```

**Comment by mrhillsman** [comment]
@kiwik we also have Linaro for ARM support; will that be better? Right now the allocation is small but we can add that in addition to VEXXHOST and if usage increases or there is need to add we can request.

**Comment by kiwik** [comment]
@mrhillsman we don't have ARM integration test request yet, so this task is not high priority.

**Comment by kiwik** [comment]
Have been addressed in #194
